File: codingame/community/rubik/rubik.m
#include <Foundation/Foundation.h>
int main(int argc, const char * argv[]) {
    int N;
    scanf("%d", &N);
    if (N > 1) {
        N = N * 6 * (N - 2) + 8;
    }
    printf("%d\n",N );
}

File: codingame/community/rubik/rubik.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>
int main()
{
    int N;
    scanf("%d", &N);
    N = N>1?6*N*(N-2)+8:1;
    printf("%d\n",N);
    return 0;
}